cypress/e2e/1-auth/sign-in.cy.ts

```diff
@@ -17,24 +17,83 @@ describe("Sign In", () => {
       firstName: `FirstName${nanoid()}`,
       lastName: `LastName${nanoid()}`,
     });
-    cy.get('[data-testid="profile-menu-item"]').click();
-    cy.get('[data-testid="logout-menu-item"]').click();
-    cy.get('[data-testid="profile-menu-item"]').should("not.exist");
     cy.visit("/sign-in");
   });
 
   it("Successful Sign In", () => {
-    cy.get('[data-testid="email"]').type(email);
-    cy.get('[data-testid="password"]').type(password);
-    cy.get('[data-testid="sign-in-submit"]').click();
+    cy.getBySel("email").type(email);
+    cy.getBySel("password").type(password);
+    cy.getBySel("sign-in-submit").click();
     cy.location("pathname").should("not.include", "/sign-in");
   });
 
   it("Successful Sign In with redirect", () => {
     cy.visit("/profile");
-    cy.get('[data-testid="email"]').type(email);
-    cy.get('[data-testid="password"]').type(password);
-    cy.get('[data-testid="sign-in-submit"]').click();
+    cy.getBySel("email").type(email);
+    cy.getBySel("password").type(password);
+    cy.getBySel("sign-in-submit").click();
     cy.location("pathname").should("include", "/profile");
   });
+
+  it("Error should be displayed if password is incorrect", () => {
+    cy.intercept('POST', '/api/v1/auth/email/login').as('login')
+    cy.getBySel('email').type(email);
+    cy.getBySel('password').type('password1');
+    cy.getBySel('sign-in-submit').click();
+    cy.wait('@login')
+    cy.getBySel("password-error").should('be.visible')
+    .and('contain.text', 'Password is not valid')
```

Comment:
Let's discuss checking for `contain.text`. I have a concern about it: If the translation file is updated it needs to change every test file where we use this translation, and this work is boring and takes a lot of time. I propose to check only `.should('be.visible')`. If you have another opinion please let me know!